screens/Settings/SettingScreen.tsx

```diff
@@ -131,6 +138,7 @@ export const SettingScreen: React.FC<SettingProps & MainRouteProps> = (
                 </ListItem.Title>
               </ListItem.Content>
             </ListItem>
+            */}
```

Comment:
Why is Inji Tour guide section commented in the settings screen?

Reply:
The Inji Tour Guide section is a place holder, and functionality is not yet done, hence commenting it, so once the functionality is implemented , we can toggle the visibility.